media-sound/mp3_check/mp3_check-1.98-r1.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="MP3 consistency checker"
+HOMEPAGE="https://sourceforge.net/projects/mp3check/"
+SRC_URI="mirror://sourceforge/mp3check/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~sparc ~x86"
+IUSE=""
+
+PATCHES=(
+	"${FILESDIR}/${P}-makefile.patch"
+)
+
+DOCS=(
+	"MILESTONE"
+	"MILESTONE.INTRO"
+	"GOALS"
+	"FOR_TESTING"
+	"THANKYOU"
+	"NOTES"
+	"MY_NOTES"
+	"WISHLIST"
+)
+
```

Comment:
I'd keep the src_compile phase (just without the now-unneeded "||die"), since then we use $CC (instead of the default setting in the Makefile, which ignores compiler selection in make.conf).